client/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components/macro";
+import React from "react";
+import img from "../../assets/hubble-eyes.jpg";
+import PropTypes from "prop-types";
+
+const ModalContainer = styled.div`
+  width: 600px;
+  background-color: #fff;
+  color: black;
+  padding: 0.8rem;
+  border-radius: 30px;
+  box-shadow: 0px 0px 15px 3px #fff;
+
+  img {
+    max-width: 100%;
+    border-radius: 25px;
+  }
+  button {
+    border: none;
+    background-color: #fff;
+    width: 20px;
+    height: 20px;
+    cursor: pointer;
+    text-align: center;
+  }
+`;
+
+const ModalHeader = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+  input {
+    border: none;
+    border-bottom: 1px solid black;
+    margin-right: 1rem;
+    padding: 0.3rem;
+    font-family: "Orbitron", Arial, sans-serif;
+  }
+`;
+
+const Modal = ({ modalOpen, setModalOpen }) => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      {modalOpen ? (
+        <ModalContainer modalOpen={modalOpen}>
+          <ModalHeader>
+            {" "}
```

Comment:
Remove this, as it is an anomaly from prettier

Reply:
Thanks, I didn´t even recognize it.. 😄